# When loading a new document, we must clear the selection, because the new document
# should have nothing selected
#
# Copyright (C) 2006 - 2007 Nokia Corporation.
# This file is distributed under the terms of GNU LGPL license, either version 2
# of the License, or (at your option) any later version.
#
diff -ru gtkhtml-3.13.91/src/htmlengine.c gtkhtml-3.13.91.mod11/src/htmlengine.c
--- gtkhtml-3.13.91/src/htmlengine.c	2007-02-15 17:52:01.000000000 +0200
+++ gtkhtml-3.13.91.mod11/src/htmlengine.c	2007-02-16 16:09:21.000000000 +0200
@@ -5000,6 +5000,8 @@
 	if (e->clue != NULL)
 		html_object_destroy (e->clue);
 
+	clear_selection (e);
+
 	g_list_foreach (e->formList, destroy_form, NULL);
 
 	g_list_free (e->formList);
Only in gtkhtml-3.13.91.mod11/src: htmlengine.c.orig
